server/src/test/java/org/candlepin/sync/ImporterTest.java

```diff
@@ -592,7 +592,7 @@ public void testReturnsSubscriptionsFromManifest() throws IOException, ImporterE
 
         Product prod = new Product("prodId", "prodTest", null);
         prod.setDependentProductIds(null);
-        File prodFile = new File("product.json");
+        File prodFile = new File("target/test/resources/upstream/product.json");
```

Comment:
Instead of loading these directly from the file path, I think the preferred way to do it is via the classpath.  E.g.

```java
ImporterTest.class.getClassLoader().getResourceAsStream("upstream/product.json");
```